packages/rest/test/acceptance/bootstrapping/rest.acceptance.ts

```diff
@@ -45,17 +46,30 @@ describe('Starting the application', () => {
         port: 0,
       },
     });
-    const server = await app.getServer(RestServer);
-    server.handler((sequence, request, response) => {
-      sequence.send(response, 'hello world');
-    });
 
-    await app.start();
-    const port = await server.get(RestBindings.PORT);
+    await startServerCheck(app);
+  });
 
-    await supertest(`http://localhost:${port}`)
-      .get('/')
-      .expect(200, 'hello world');
-    await app.stop();
+  it('starts an HTTP server', async () => {
+    const app = new RestApplication();
+    app.bind(RestBindings.PORT).to(0);
+    await startServerCheck(app);
   });
 });
+
+// Helper function to spin up the application instance and assert that it
+// works.
+async function startServerCheck(app: Application) {
+  const server = await app.getServer(RestServer);
+  server.handler((sequence, request, response) => {
```

Comment:
Since we know that `RestApplication` has only a single server instance, I'd like helper methods like `handler` and `sequence` to be added to `RestApplication`.

To me, that's the most important part about RestApplication, because it brings us back to the easy-to-use design we had before the REST component was extracted from core.

```ts
const app = new RestApplication();
app.handler((sequence, request, response) => {
  sequence.send(response, 'hello world');
});
```

Since this is an additional change, I am ok to make it later, in a follow-up pull request.